node/network/protocol/src/request_response/mod.rs

```diff
@@ -243,13 +248,13 @@ impl Protocol {
 		}
 	}
 
-	/// Get the protocol name of this protocol, as understood by substrate networking.
-	pub fn into_protocol_name(self) -> Cow<'static, str> {
-		self.get_protocol_name_static().into()
+	/// Fallback protocol names of this protocol, as understood by substrate networking.
+	pub fn get_fallback_names(self) -> Vec<Cow<'static, str>> {
```

Comment:
I am guessing this ensures backwards compatibility?

Reply:
Yes, I think we can at some point obsolete protocol names starting with "/polkadot/..."